File: spiritual-site/test_pdf.py
import PyPDF2

def get_text():
    pdf_path = r'C:\Users\alind\Documents\Brianna_Wiest_-_101_eseuri_care_iti_vor_schimba_modul_de_a_gandi.pdf'
    text = ""
    with open(pdf_path, 'rb') as f:
        reader = PyPDF2.PdfReader(f)
        for i in range(10): # read first 10 pages
            text += reader.pages[i].extract_text() + "\n"
    print(text[:3000])

if __name__ == "__main__":
    get_text()

File: public/spiritual-site/test_pdf.js
const fs = require('fs');
const pdf = require('pdf-parse');

let dataBuffer = fs.readFileSync('C:\\Users\\alind\\Documents\\Brianna_Wiest_-_101_eseuri_care_iti_vor_schimba_modul_de_a_gandi.pdf');

pdf(dataBuffer).then(function(data) {
    // text of the first 5000 characters
    console.log(data.text.substring(0, 5000));
}).catch(console.error);
